docs/attribution.md

```diff
@@ -66,10 +66,10 @@ More details can be found on the Natural Earth [Terms of Use](http://www.natural
 
 `source:whosonfirst`
 
-Crediting [Who's On First](https://whosonfirst.mapzen.com) is recommended and linking back to their License is required.
+Crediting [Who's On First](https://www.whosonfirst.org) is recommended and linking back to their License is required.
 
 ```
-Data from Who's On First. <a href="https://whosonfirst.mapzen.com#License">License</a>
+Data from Who's On First. <a href="https://www.whosonfirst.org">License</a>
```

Comment:
This URL is better to link to: https://www.whosonfirst.org/docs/licenses/.

Though this is the ultimate truth: https://github.com/whosonfirst/whosonfirst-sources/blob/master/sources/README.md

Reply:
Data from Who's On First. <a href="https://www.whosonfirst.org">License</a> and <a href="https://github.com/whosonfirst/whosonfirst-sources/blob/master/sources/README.md">Sources</a>